### DISCUSSION TITLE
problem about nextflow tower

### DISCUSSION THREAD

**Discussion by sqwwww** [nextflow-io/nextflow#2664] [Q&A]
Hi! I met a question when used nextflow tower.
![image](https://user-images.githubusercontent.com/98629662/155076512-04ea3b56-8550-4b1a-add6-12a2ca6cd74c.png)

when I tried to download it following the instruction, it failed because https://www.nextflow.io/releases/v20.10.6/nextflow-20.10.6-one.jar didn't exist
![image](https://user-images.githubusercontent.com/98629662/155076204-13262e00-88f1-45fa-878b-45a61bf4e584.png)

**Comment by pditommaso** [comment]
I suspect it's a network problem on your side. Are you using a proxy to access the internet?

**Reply by sqwwww** [reply]
yes, I'm using a proxy, so the disconnectivity to the https://www.nextflow.io/releases/v20.10.6/nextflow-20.10.6-one.jar  is probably caused by the network problem? sorry for the bothering, but is there another way for me to get it?

**Reply by sqwwww** [reply]
when I searched the website https://www.nextflow.io/releases/v20.10.6/nextflow-20.10.6-one.jar in the google chrome directly, it shows "page not found", while I can normally get other websites. Is there a probability that maybe this website is not correct or move to a new address?
